README.md

```diff
@@ -1,3 +1,5 @@
+# This action's purpose can now be fulfilled with the latest version of [actions/cache](https://github.com/actions/cache).
```

Comment:
This action still fulfills a slightly different purpose.  The cache-hit output was available when this action was created, but it still requires downloading the cache to determine if there was a hit or not.  The purpose of this action is determine whether a hit exists without having to download all the artifacts.

Does that address this, or am I missing something?